pkg/compiler/analysis.go

```diff
@@ -235,10 +234,6 @@ func (c *codegen) fillDocumentInfo() {
 	fset := c.buildInfo.config.Fset
 	fset.Iterate(func(f *token.File) bool {
 		filePath := f.Position(f.Pos(0)).Filename
-		filePath, err := filepath.Rel(c.buildInfo.config.Dir, filePath)
```

Comment:
Usually, absolute paths in debug files are to be avoided. Maybe we can distinguish between contract files (that shouldn't have this problem) and interops?

Reply:
@fyrchik told me about that also, but I wasn't sure whether it's OK to combine both relative and absolute file formats.

Fixed, could you check?

Reply:
Oh, not fixed yet.

Reply:
Or maybe always store absolute path if relative can't be derived (`err` handling essentially)?

Reply:
Yes, I like this solution the most, implemented.